FHIR_README.md

```diff
@@ -0,0 +1,78 @@
+![img](./public/images/openemr-rest-api.png)
+
+### Overview
+
+Easy-to-use JSON-based REST API for OpenEMR FHIR. All code is done in classes and separate from the view to help with codebase modernization efforts. 
+
+### Prerequisite
+Enable this FHIR service in OpenEMR menu: Administration->Globals->Connectors->"Enable FHIR Provider Client Service" Select HAPI FHIR or Smart on FHIR, then Save
```

Comment:
Selecting HAPI or SMART on FHIR isn't necessary

Reply:
Agree with @yashrajbothra 
Note you just do the same as the API here:
```
Enable this API service in OpenEMR menu: Administration->Globals->Connectors->"Enable OpenEMR REST API"
```

Also not that like the API, these endpoints can be called internally (see that section in API doc)

Reply:
Fixed